docs/projects/P2/1_teamprocess.md

```diff
@@ -0,0 +1,172 @@
+
+You should use this team repository for all your development, and be sure to use good development practices, including keeping your commits cohesive and your commit messages informative. The same resources mentioned above provide more details on our guidelines.
+
+We will be grading you on how well you follow the process we used for Project 1 and utilize this Project board:
+
+- Create issues for feature improvements or bug-fixes
+- When creating issue, assign team members and tag with appropriate labels
+- Create a pull request and reference the issue it will be resolving
+- Provide feedback to pull requests
+- Use a Kanban board to track your workflow
+
+For this project, you will create issues or pull requests **directly in your team repository**. We will use your commit history and issue/pull request activity on Github to assess both your teamwork process and each member’s individual contributions towards the project. It is **not** acceptable for one person to commit all the work after synchronizing through other means.
+
+For each code-based deliverable, we will look at a snapshot of your repository at the deadline. This will often be done by asking you for a screenshot or commit link submitted via Gradescope.
+
+
+### Teamwork Self-Assessment
+
+As the course moves into team-based projects, we will be releasing weekly self-assessments on Canvas, each worth one participation point. These are meant to be short surveys should not take more than a few minutes that may help you reflect on your team's process and how to improve it. The first one will be released on **Friday, January 27th** and due the following **Friday, February 3rd** (both at 11:59pm).
+
+
+## Main Deliverables
+
+### Teamwork Contract (40 pts)
+
+When working with a team, it is important to discuss each team member’s background, and establish common expectations of the team. Miscommunication or the general lack of communication are often the most common causes of team conflict. 
+
+!!! note "Team Conflict Example"
+    A common conflict in working style is when there are team members who always want to get a headstart on their work, while there are team members who are fine with doing work a few days before the deadline. It causes panic in the former team members, while the latter team members feel frustrated as to why they are being rushed. 
+
+As such, your first process task of the semester will be creating a teamwork contract with your teammates. It is a **1-2 page** document (soft cap) containing information that all teammates agree to follow. You should work on the contract **with all members present**. Additionally, **it is more important that you only include statements that the team will adhere to than it is to fulfill the length requirement** (quality over quantity!)
```

Comment:
I think we should say something along the lines of not needing to write complete sentences, and encouraging bullet points instead of long prose